docs/wiki/framework/events-framework.md

```diff
@@ -47,7 +47,7 @@ MenuType: 0 = Interaction, 1 = Self Interaction
 |`ace_interactMenuOpened` | [_menuType] | Local | Listen | Interaction Menu Opened
 |`ace_interactMenuClosed` | [_menuType] | Local | Listen | Interaction Menu Closed
 
-### 2.4 Logistics (`ace_cargo`)
+### 2.4 Cargo (`ace_cargo`)
```

Comment:
careful, this will make some potential incoming links kaputt!
Could you please do a string search for pages that link to that section (if there are any) and rewrite that link?

Reply:
0 matches in 5251 files it seems 👍

Reply:
Only thing that would make sense to link here would be https://ace3mod.com/wiki/framework/cargo-framework.html#21-listenable
But as you can see it has it's own table. Maybe we could just link here instead of having a copy of the table?
Would be bad UX tho.

Reply:
I had the same irritation. Would be great if the events site could aggregate all the event sections from the single framework pages.

Buuuut I dont think reworking the entire structure is in the scope of this PR ;)

Reply:
Could write a script to aggregate and link them all across. I like them on framework pages, each component gets one and is harder to miss something outdated.

Not in scope of this PR as you said.